mmaction/models/backbones/tanet.py

```diff
@@ -0,0 +1,108 @@
+from copy import deepcopy
+
+import torch.nn as nn
+from torch.utils import checkpoint as cp
+
+from ..common import TAM
+from ..registry import BACKBONES
+from .resnet import Bottleneck, ResNet
+
+
+class TABlock(nn.Module):
+    """Temporal Adaptive Block (TA-Block) for TANet.
```

Comment:
Paper link should be added like https://github.com/open-mmlab/mmaction2/blob/master/mmaction/models/backbones/resnet3d_slowfast.py#L366-L367